src/main/java/com/google/sps/servlets/utils/CommonWordsRetriever.java

```diff
@@ -20,15 +20,14 @@
 
 /** Class provides commonly used words that should be ignored within the word map */
 public class CommonWordsRetriever {
-  private static List<String> commonWordsList;
+  // TODO: Parse text file for common words
+  private static final List<String> commonWordsList =
+      new ArrayList<>(
+          Arrays.asList(
+              "the", "it", "at", "and", "to", "we", "can", "are", "of", "is", "am", "i", "this",
+              "was"));
 
   public static List<String> getCommonWords() {
-    // TODO: Get words by parsing text file
-    commonWordsList =
-        new ArrayList<>(
-            Arrays.asList(
-                "the", "it", "at", "and", "to", "we", "can", "are", "of", "is", "am", "i", "this",
-                "was"));
     return commonWordsList;
```

Comment:
Here we should return a copy of the `commonWordsList`.

I.e. `return new ArrayList<String>(commonWordsList)`.

The reason is that ArrayList is mutable and by gaining access to the actual list instance being used by this class, consumers could potentially change the elements of the list.

Alternatively you could also use a `List` type that is immutable by design like `ImmutableList`.